judgels-frontends/raphael/src/routes/uriel/contests/single/submissions/Bundle/GradingTag/GradingTag.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from 'react';
+import { Tag, Intent } from '@blueprintjs/core';
+import { Grading, Verdict } from 'modules/api/sandalphon/submissionBundle';
+
+export interface GradingTagProps {
+  grading?: Grading;
+}
+
+export const GradingTag: FunctionComponent<GradingTagProps> = ({ grading }) => {
```

Comment:
Please call this `VerdictTag`, similar to programming's, and accept `verdict`.

The caller should not show any tag if the grading is missing.